cluster/log-dump/log-dump.sh

```diff
@@ -138,26 +139,27 @@ function copy-logs-from-node() {
 function save-logs() {
     local -r node_name="${1}"
     local -r dir="${2}"
-    local files="${3}"
+    local save_files="${3}"
     local opt_systemd_services="${4:-""}"
     local on_master="${5:-"false"}"
 
     files="${files} ${extra_log_files}"
```

Comment:
This is where shellcheck currently throws you an error; `files` is assigned as a string but used as an array.